sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Config/EventGridExtensionConfigProvider.cs

```diff
@@ -50,7 +50,7 @@ internal class EventGridExtensionConfigProvider : IExtensionConfigProvider,
             _collectorFactory = collectorFactory;
             _httpRequestProcessor = httpRequestProcessor;
             _loggerFactory = loggerFactory;
-            _diagnosticScopeFactory = new DiagnosticScopeFactory(DiagnosticScopeNamespace, ResourceProviderNamespace, true, false);
```

Comment:
Just to make sure I understand correctly: this is an internal API, so needing to change tests won't have any downstream customer impact, correct?

Reply:
This is not a test. I think we are just removing the default parameter?

Reply:
discussed offline: removing this has no effect at all since the only activity we create with `_diagnosticScopeFactory` is consumer and they are never suppressed

Reply:
Thanks so much, @lmolkova for walking me through all these changes!